main.go

```diff
@@ -198,22 +237,60 @@ func regoSyntax(p string) int {
 
 	if compiler.Compile(modules); compiler.Failed() {
 		for _, err := range compiler.Errors {
-			fmt.Fprintln(os.Stderr, err)
+			_, _ = fmt.Fprintln(os.Stderr, err)
 		}
 		return 1
 	}
 
 	return 0
 }
 
+func initOPA(ctx context.Context, configFile string) (*sdk.OPA, error) {
+
+	buf, err := os.Open(configFile)
+	if err != nil {
+		return nil, err
+	}
+
+	defer func() {
+		if err = buf.Close(); err != nil {
+			log.Fatal(err)
+		}
+	}()
+
+	options := sdk.Options{
+		Config: bufio.NewReader(buf),
+	}
+
+	opa, err := sdk.New(ctx, options)
+	if err != nil {
+		return nil, err
+	}
+
+	return opa, nil
+}
+
+func normalizeAllowPath(path string, useConfig bool) string {
+
```

Comment:
Good question. How does that even look as a path? `/foo["com.example.foo"]/allow`?

Reply:
I'd think `/foo/com.example.foo/allow`? But let's try if it work that way 😉

Reply:
Ah, of course! That complicates things, but I'll give it a shot :)

Reply:
Hehe, and this is just covering the transformation in one direction.. I'm sure it can be done nicer, but maybe better to keep it simple as to not clutter up the whole file with this, or WDYT? It was essentially just meant to help catch an easy mistake, especially as their is a default value provided for `allowPath` which would not work with the SDK unless transformed.

```go
func normalizeAllowPath(path string, useConfig bool) string {
	if useConfig && strings.HasPrefix(path, "data") {
		path = strings.TrimPrefix(path, "data")
		newPath := ""

		var inBrackets bool
		for _, character := range path {
			c := string(character)
			if c == "[" {
				inBrackets = true
				newPath += "/"
				continue
			}
			if inBrackets && c == "]" {
				inBrackets = false
				newPath += "/"
				continue
			}
			if !inBrackets && c == "." {
				newPath += "/"
				continue
			}
			if c != "\"" {
				newPath += c
			}
		}

		return newPath
	}
}
```

Reply:
On the topic of `allowPath`, it drives me absolutely insane that it's the only command line argument that uses camel case instead of the usual `allow-path` 🤦  But.. backwards compatibility 🙃